packages/grid/_modules_/grid/models/gridColumnHeaderClass.ts

```diff
@@ -0,0 +1,11 @@
+import { GridColumnHeaderParams } from './params/gridColumnHeaderParams';
+
+/**
+ * A function used to process headerClassName params.
+ */
+export type GridColumnHeaderClassFn = (params: GridColumnHeaderParams) => string;
+
+/**
+ * The union type representing the [[GridColDef]] column header class type.
+ */
+export type GridColumnHeaderClassNamePropType = string | GridColumnHeaderClassFn;
```

Comment:
Why is inlining the type not enough? It seems that we never use it more than once.
```suggestion
```